### DISCUSSION TITLE
The Eduverse Hackathon (2021)

### DISCUSSION THREAD

**Discussion by mrawhite** [AngellXR/eduverse#6] [Q&A]
Let's start our discussion of the Hackathon here. I'm going to free-write some ideas and we can flesh it out as we go along. This is in addition to the planning we have done so far in our [Figma document](https://www.figma.com/file/TX7WYdWcIVsz9zfSsU55tF/EnterTheEduVerse-planning?node-id=0%3A1). 

### Purpose of the Eduverse Hackathon
Many educators and students are not quite aware of WebXR and it's potential for learning. This hackathon is an attempt to change that. We invite learners and educators of all ages to submit an immersive experience utilizing a variety of web-based resources based on a theme. Submissions could be hosted on [eduverse.io](https://www.eduverse.io/) and the [AngellXR Showcase](https://xrshowcase.xyz/). (_Forgive me for the lack of content on Eduverse. I has ideas for the website, just not the time. Slowly but surely it will flesh out._)

### Eduverse Hackathon Theme
**The History of....**
☝️ We leave this option wide open for users to interpret. I personally was motivated by this topic when thinking about [Ishkur's Guide to Electronic Music](https://music.ishkur.com/#). Yes, I was, and remain an EDM freak. When thinking of this hackathon I immediately thought of how awesome it would be to recreate this guide in an immersive environment (minus the profanities) that showcased with video, audio, and imagery the evolution of EDM music. Another inspirational website is Keith Chan's [AnVRopomotron](https://keithcchan.com/2020/02/anvropomotron-the-origin.html), and really serves as the pinnacle of what can be created with A-Frame. All that being said, participants are encouraged to create any and all history-themed immersive experiences they want. History of dogs? Sure. History of ice cream? Why not? Celebration of Juneteenth and emancipation? Hell yes! Anything and everything creators can think of. And what better creators of unique and invigorating content than students? 

### How can I create these experiences?
We will share a variety of easy-to-use platforms that are device-agnostic, safe, and secure for all participants to create. This includes A-Frame, FrameVR, Hubs by Mozilla, Jell.App, and....I'm not sure, add them in a comment below! Anything that a) doesn't require a login to create (or at least adheres to [FERPA/COPPA/CIPA](https://www.spps.org/cms/lib/MN01910242/Centricity/Domain/11270/OverviewofCIPACOPPAandFERPA12.2015.pdf) requirements), b) is relatively easy to create with, c) is easy to share out. If it means creating a dumping ground for users to paste their A-Frame code so we can create Glitches based on that content, so be it. Whatever is the easiest and quickest method to create. Low barrier of entry, no-code to low-code experiences is what we're aiming for. 

### Who is our target audience?
Learners and educators of all ages, really. Obviously kindergartners won't be coding entity-components, but they could potentially share something they created with Cospaces....maybe? I'm biased towards K-12 education, but colleges and universities are welcome to participate as well. 

This is a start! Comment below on recommendation, critiques, ideas, or anything that comes to mind that can make this the first of its kind for the education space. We're heading into new territory, people! I hope you brought your towel.

**Comment by mrawhite** [comment]
Let me add a suggestion to this post:

If you are interested or invested in WebXR in EDU and want to be added as a contributor to eduverse.io, please let me know!

**Comment by planetvoodoo** [comment]
In the spirit of the open web, decentralization and self-empowerment - maybe the onus should be on the student? Where they're tasked with building something which not only helps cement knowledge for themselves, but which can also be shared as an educational tool with other students to help others learn through this creative exercise. With each person being unique, we often take an interest in a specific area/field/niche - handing 'power to the mob' in this case, may encourage specialized, shareable educational experiences. Being invested in the building of your own curriculum may be the key to maintaining engagement. The model seems to work for Wikipedia at least!

**Reply by mrawhite** [reply]
I agree 100% with what you wrote here. Students should be given choice on how they wish to build their experience. I think having a theme will be good though. The theme I proposed was history of something, could be the history of the grilled cheese sandwich. Any other themes you think would interest students?

**Comment by heathervt** [comment]
https://courses.cs.washington.edu/courses/cse481v/21sp/demo-day/
"This year was unique in that all of our projects were built in WebXR for the Oculus Quest 2 or the Magic Leap One."
